website/common/script/content/faq.js

```diff
@@ -14,7 +14,11 @@ for (let i = 0; i <= NUMBER_OF_QUESTIONS; i++) {
   let question = {
     question: t(`faqQuestion${i}`),
     ios: t(`iosFaqAnswer${i}`),
-    web: t(`webFaqAnswer${i}`),
+    web: t(`webFaqAnswer${i}`, {
+      //  TODO: Need to pull these values from nconf
```

Comment:
techassistance@habitica.com isn't correct. It should be admin@habitica.com.

I've checked the use of the env variables through all these changes and they're all good.